tests/frontend/test_circuitspecs_chip0.py

```diff
@@ -317,17 +320,17 @@ def test_mach_zehnder_interferometers(self, tol):
             ops.S2gate(0.5, 0) | (q[1], q[3])
 
             # corresponds to MZgate(phi, theta) on modes [0, 1]
-            ops.Rgate(phi) | q[0]
+            ops.Rgate(phi % (2*np.pi)) | q[0]

```

Comment:
how come you used `np.mod` elsewhere?

Reply:
Oh. Just me not being consistent. I can change everything over to `np.mod`.

Reply:
Done